src/index.js

```diff
@@ -38,7 +38,7 @@ export default function configureStore (middlewares = []) {
           actions.push(action)
 
           for (let i = 0; i < listeners.length; i++) {
-            listeners[i]()
+            listeners[i](action)
```

Comment:
Should this action be passed in here after removing the `type` property?

Reply:
Unless there's a huge demand for it, I'd leave the `type` property, it gives more rich info to any listeners in test.